sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -108,6 +109,7 @@
         new TableFunction(new FunctionCall(QualifiedName.of("empty_row"), Collections.emptyList()))
     );
 
+    @Inject
```

Comment:
I think now that it's injected into the InternalViewInfoFactory would be confusing to also manually instantiate it here: https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/analyze/Analyzer.java#L136 Can it be also injected?

Reply:
Would make testing a bit more cumbersome.

I'd tend to leave it as it is since the `RelationAnalyzer` instance is quite small.